src/main/java/forestry/api/storage/IBackpackInterface.java

```diff
@@ -39,7 +40,7 @@
 	 * @param itemStack   The itemStack that the backpack should accept.
 	 *                    {@link OreDictionary#WILDCARD_VALUE} can be used for meta value.
 	 */
-	void addItemToForestryBackpack(String backpackUid, ItemStack itemStack);
+	void addItemToForestryBackpack(String backpackUid, @Nullable ItemStack itemStack);
```

Comment:
@mezz Do you think allowing null for the api methods is a good idea ?

Reply:
`ItemStack` should almost never be null, it has to be a very special case.